### DISCUSSION TITLE
Error saat update pip

### DISCUSSION THREAD

**Discussion by npmitaart** [teamalgoritma/community#10] [Q&A] [answered]
Halo teman-teman, saya mau izin nanya terkait update pip di Python. Kemarin saya sedang mencoba install beberapa library di Python. Trus ada peringatan kalau pip versionnya sudah ada yang terbaru.

Saya sudah coba ikuti untuk update
```
pip install --upgrade pip
```

dan error yang muncul
```
ValueError: Unable to find resource t64.exe in package pip._vendor.distlib
```

tadi saya sudah coba beberapa cara dan uninstall pip tapi masih error, mohon bantuannya teman-teman. Terima kasih :D

![error_update_pipv22](https://user-images.githubusercontent.com/50654221/158322424-6a00890f-f52b-4694-9cd5-572270fad8e9.png)

**Comment by onlyphantom** [comment, marked as answer]
Solution probably involves uninstalling pip and then reinstalling it again:

```
python -m pip uninstall pip
python -m ensurepip
python -m pip install -U pip
```

If that didn't work, go ahead and uninstall `pip` and `setuptools`

```
python -m pip uninstall pip setuptools
```

Then reinstall `pip` again directly from the repository:
```
# ubuntu / debian
sudo apt install python3-pip
```

Related discussion: https://github.com/pypa/pip/issues/8450

**Reply by npmitaart** [reply]
Thank you Kak Sam! 

```
python -m pip uninstall pip setuptools
```
and reinstalling pip from the repository worked for me!
